test/controllers/beneficial.owner.statements.spec.ts

```diff
@@ -290,10 +292,9 @@ describe("BENEFICIAL OWNER STATEMENTS controller", () => {
       expect(mockSaveAndContinue).not.toHaveBeenCalled();
     });
 
-    // TODO - UPDATE THESE TESTS, ADD PARAMETERS TO BENEFICIAL_OWNER_DELETE_WARNING_PAGE
     test(`redirects to ${config.BENEFICIAL_OWNER_DELETE_WARNING_PAGE}
               page with NONE_IDENTIFIED as beneficial owners statement type`, async () => {
-      mockIsActiveFeature.mockReturnValueOnce(true); // For FEATURE_FLAG_ENABLE_REDIS_REMOVAL
+      mockIsActiveFeature.mockReturnValue(true); // For FEATURE_FLAG_ENABLE_REDIS_REMOVAL
```

Comment:
changing from mockReturnValueOnce to mockReturnValue will mean that teh mock will always return the true value for each call to it. It might be better to have separate mockReturnValueOnce calls and comment what they are for